lib/baserequest.go

```diff
@@ -51,6 +52,11 @@ func (c *Conn) DoCommand(method string, url string, args map[string]interface{},
 		}
 	}
 
+	// uncomment this to print out the request that hits the wire
+	//   (requires net/http/httputil)
+	//reqbuf, err := httputil.DumpRequest(req.Request, true)
+	//log.Println(fmt.Sprintf("\n========= req:\nURL: %s\n%s", req.URL, bytes.NewBuffer(reqbuf).String()))
```

Comment:
Yes, it is very useful since it prints out the request URL and body that go out on the wire.  I was trying to think of a better way to hide it behind a compile-time debug flag or something.  Do you have any suggestions?